app/controllers/boarding_points_controller.rb

```diff
@@ -0,0 +1,51 @@
+class BoardingPointsController < ApplicationController
+  before_action :find_boarding_point_by_id, only: %i[edit update destroy]
+
+  def index
+    @boarding_points = BoardingPoint.all.order('id')
+  end
+
+  def new
+    @boarding_point = BoardingPoint.new
+  end
+
+  def edit; end
+
+  def create
+    @boarding_point = BoardingPoint.new(boarding_point_params)
+
+    if @boarding_point.save
+      redirect_to action: 'index', status: :created
+    else
+      render :new, status: :unprocessable_entity
+    end
+  end
+
+  def update
+    if @boarding_point.update(boarding_point_params)
+      redirect_to action: 'index', status: :see_other
+    else
+      render :edit, status: :unprocessable_entity
+    end
+  end
+
+  def destroy
+    @boarding_point.destroy
+    redirect_to action: 'index', status: :see_other
```

Comment:
I am not aware of this particular status :`see_other`, can you please explain what does it mean? 🤔

Reply:
Apu, When the code `status: :see_other` is used in the `redirect_to` method in Ruby on Rails, it tells the browser to redirect the user to a different URL, typically specified by the action and controller parameters. In this case, the user will be redirected to the `index` action of the current controller.